python/pyspark/sql/pandas/conversion.py

```diff
@@ -20,9 +20,10 @@
 
 from pyspark.rdd import _load_from_socket
 from pyspark.sql.pandas.serializers import ArrowCollectSerializer
-from pyspark.sql.types import IntegralType
 from pyspark.sql.types import ByteType, ShortType, IntegerType, LongType, FloatType, \
-    DoubleType, BooleanType, MapType, TimestampType, StructType, DataType
+    DoubleType, BooleanType, MapType, TimestampType, StructType, DataType, \
+    IntegralType, _is_datatype_with_udt
+from pyspark.sql.pandas.types import _deserialize_pandas_with_udt
```

Comment:
these two `_` prefixed functions are used outside their files. should we remove the `_` prefix, because they are not private anymore.

Reply:
`git grep _make_type_verifier`, there are other use cases which a function starts with `_` but is used outside where they are defined.